### PR TITLE
Implement basic tests

### PR DESCRIPTION
**Pull request by sgill2**
Adds some tests to make sure the package is working properly. Added in this PR is tests to make sure the com and rotations are being applied correctly, and tests to make sure a very basic ncmc simulation runs (with rotational moves too).

Also from later comments:
–adds a blues/data directory which contains some systems for testing purposes
–changes url in setup.py to specify new url

### CODE REVIEW

**Comment by davidlmobley** [comment]
LMK when you're ready for review on this.

**Comment by sgill2** [comment]
@davidlmobley For Travis tests what's the appropriate way to refer to file paths for my scripts. Is it relative from the test*.py directory, or does it use some other scheme? Besides fixing up the file paths it should be ready to review.

**Comment by davidlmobley** [comment]
@sgill2 - the tests get *installed* with the package by Travis (e.g. by `setup.py`) so you need to (a) have your setup.py install your data files as well as the rest of the package, and (b) then locate the files within the package. This can be nontrivial. 

Nathan had to deal with this recently too so he may be able to advise. But see also `smarty`, where the setup.py installs a `data` directory (https://github.com/open-forcefield-group/smarty/blob/master/setup.py) and there is a utility `get_data_filename` that the tests use to help find data files within the package. YOu may want to do something similar. https://github.com/open-forcefield-group/smarty/blob/master/smarty/tests/test_forcefield.py#L204

**Comment by sgill2** [comment]
@davidlmobley so I think I got everything working. It says there's a failing check but that's actually to be expected with this build (and what PR #27 addresses).

**Comment by davidlmobley** [comment]
@sgill2 - so, "WIP" is ready to be removed from the title and this is ready to merge once I review? Or did you want to finalize #27 first so we can see if this passes tests?

**Comment by davidlmobley** [comment]
Also, slight pet peeve - I like titles of PRs to express complete ideas, like "implement basic tests", not "basic tests". :)

**Comment by sgill2** [comment]
@davidlmobley I've made and applied changes you've suggested above.

File: blues/data/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Files used for unit testing purposes.
+circle.xml is a force field.
```
Comment (davidlmobley) [inline blues/data/README.md]: Right, well, I gathered it was a force field, but one might wonder, "what does this have to do with a circle?", or "what force field??" For example, "circle.xml is a simple force field used for unit testing purposes which applies a harmonic potential to (whatever) (something having to do with a circle)," or whatever it is.

File: blues/data/README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+squareB2 is a simple square model system.
```
Comment (davidlmobley) [inline blues/data/README.md]: Again, exactly what? 

Imagine I'm reading this and wanting to know if I can use your file to design a new test to test some other functionality. I have to know more than that it's a square. For example, "squareB2 is a square model system containing N(??) particles arranged in a (size x size) square. It is used for unit testing purposes, particularly to ensure that..." or some such. Remember, the goal here is to make it so I can know what things are here for without having to read your code.

Comment (davidlmobley) [inline blues/data/README.md]: (So, when you write things, you have to try to imagine yourself as a user who knows nothing about this project and you are trying to make sense of the files and how everything fits together from only the README information provided.)

**Comment by sgill2** [comment]
One last change before merging is that I changed my tests to use the openmmtools alanine dipeptide system instead, so there's no need for a data folder.
